.azure-pipelines/scripts/setup-integration-chaincode.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+DIR=$(pwd)
+CHAINCODE_DIR=$DIR/.azure-pipelines/resources/chaincode
```

Comment:
what's this file doing? it needs a comment, at least

Reply:
Yeah I'll add a comment. Since the code written in the PR is not merged yet we need to use the local files. This is updating the go.mod file to point to the local files, vendor them in and then remove that pointer. This means that the chaincode can be built by the in the docker image setup by the integration tool using the vendored files

Reply:
If we didn't do this it would use the version found on github in master. If we left the pointer to the local files in permanently the build process in docker would cry as the files don't exist on there